client/Cargo.toml

```diff
@@ -40,7 +40,10 @@ appveyor = { repository = "bluejekyll/trust-dns", branch = "master", service = "
 coveralls = { repository = "bluejekyll/trust-dns", branch = "master", service = "github" }
 
 [features]
-default = ["openssl"]
+default = ["dnssec-openssl"]
```

Comment:
@briansmith in regards to this:

> The reason I want to do this is because I want to advocate for trust-dns-resolver to be the default DNS implementation (at least on some Unixy platforms), and lots of users of the resolver will not only be uninterested in DNSSEC, but they are even actively opposed to it (e.g. web browsers).

While working on this change, I was getting confused around some of these options. You mentioned a `dnssec` module, I'm starting to get crate fatigue with the number we have already. But, I did agree, and because of the aliasing rules with `dependecies` and `features` in `cargo`, I decided to put this into the vernacular:

- `dnssec-<impl>`: where impl is `\*ring\*` | `openssl` today.
- `dnssec`: which can be used as a global flag to remove anything DNSSec related.